### PR TITLE
Add instruction for setting archival node without nearup

### PR DESCRIPTION
**Pull request by janewang**

### CODE REVIEW

**Comment by janewang** [comment]
Master branch seems to have a broken link
http://localhost:5000/docs/develop/contracts/debug-locally
  [404] https://docs.near.org/docs/develop/contracts/sandbox

File: docs/develop/node/archival/run-archival-node-without-nearup.md [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+---
+
+### Choosing your `nearcore` version
+
+When building your NEAR node you will have two branch options to choose from depending on your desired use:
+
+- `master` : _(**Experimental**)_
+  - Use this if you want to play around with the latest code and experiment. This branch is not guaranteed to be in a fully working state and there is absolutely no guarantee it will be compatible with the current state of *mainnet* or *testnet*.
+- [`Latest release branch`](https://github.com/near/nearcore/releases) : _(**Stable**)_
+  - Use this if you want to run a NEAR node for *mainnet* or *tesnet*. This version is used by validators and other nodes and is fully compatible with the current state of *mainnet* or *testnet*.
```
Comment (bowenwang1996) [inline docs/develop/node/archival/run-archival-node-without-nearup.md:39]: This is not completely true. When we make a release, we first release a rc version for testnet and then later make that release stable. So for testnet, running the latest stable version does not always work. We should change it to running the latest rc version

**Comment by thisisjoshford** [comment]
> Master branch seems to have a broken link
> http://localhost:5000/docs/develop/contracts/debug-locally
> [404] https://docs.near.org/docs/develop/contracts/sandbox

#765 fixed this. You can now merge master into this branch to resolve your failing CI @janewang . :)

**Comment by janewang** [comment]
Hi @chefsale, could you provide some feedback for this PR? Cheers and thanks!

**Review by bowenwang1996 — APPROVED** [review]

**Pull request merged**